### PR TITLE
JFG-747: Remove MonitoringFeatureComparator and WorkloadFeatureComparor

### PR DESCRIPTION
**Pull request by anton-antonenko**

### CODE REVIEW

File: chassis/configuration/configuration/reporter/session.comparison.conf.xml [hunk 1]
```diff
@@ -51,21 +50,6 @@
         <property name="databaseService" ref="databaseService"/>
     </bean>
 
-    <bean id="workloadFeatureComparator"
-          class="com.griddynamics.jagger.engine.e1.sessioncomparation.workload.WorkloadFeatureComparator">
-        <property name="workloadDecisionMaker">
-            <bean class="com.griddynamics.jagger.engine.e1.sessioncomparation.workload.ThroughputWorkloadDecisionMaker">
```
Comment (dlatnikov) [inline chassis/configuration/configuration/reporter/session.comparison.conf.xml]: ThroughputWorkloadDecisionMaker - why is it not deleted from the code base as well? It is not used any more

\+ WorkloadDecisionMaker
\+ StdDevMonitoringParameterDecisionMaker


please, review the whole:
/chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/sessioncomparation/monitoring

File: chassis/configuration/configuration/reporter/session.comparison.conf.xml [hunk 2]
```diff
@@ -77,13 +61,6 @@
 
     <!-- Monitoring -->
     <!-- Deprecated. Not used starting from 1.2.5 -->
```
Comment (dlatnikov) [inline chassis/configuration/configuration/reporter/session.comparison.conf.xml]: please, remove comment

File: chassis/configuration/configuration/reporter/session.comparison.conf.xml [hunk 3]
```diff
@@ -77,13 +61,6 @@
...
-    <bean id="monitoringFeatureComparator"
-          class="com.griddynamics.jagger.engine.e1.sessioncomparation.monitoring.MonitoringFeatureComparator">
-
-        <property name="monitoringParameterDecisionMaker" ref="stdDevMonitoringParameterDecisionMaker"/>
-        <property name="monitoringSummaryRetriever" ref="monitoringSummaryRetriever"/>
-        <property name="sessionFactory" ref="sessionFactory"/>
-    </bean>
     <bean id="stdDevMonitoringParameterDecisionMaker"
           class="com.griddynamics.jagger.engine.e1.sessioncomparation.monitoring.StdDevMonitoringParameterDecisionMaker">
```
Comment (dlatnikov) [inline chassis/configuration/configuration/reporter/session.comparison.conf.xml]: in this XML
following code is useless
```
 <alias name="defaultSessionComparator" alias="sessionComparator"/>
    <bean id="defaultSessionComparator"
          class="com.griddynamics.jagger.engine.e1.sessioncomparation.ConfigurableSessionComparator">
        <property name="comparatorChain">
            <ref bean="comparators_${chassis.decision.maker.with.limits.is.only.decision.maker}"/>
        </property>
        <property name="decisionMaker" ref="worstCaseDecisionMaker"/>
    </bean>
    <bean id="worstCaseDecisionMaker"
          class="com.griddynamics.jagger.engine.e1.sessioncomparation.WorstCaseDecisionMaker"/>

    <util:list id="comparators_true">
        <ref bean="limitsBasedFeatureComparator"/>
    </util:list>

    <util:list id="comparators_false">
        <ref bean="workloadFeatureComparator"/>
        <ref bean="limitsBasedFeatureComparator"/>
    </util:list>
```

\+ chassis.decision.maker.with.limits.is.only.decision.maker will be always true => can be deleted

File: chassis/spring.schema/src/main/java/com/griddynamics/jagger/xml/beanParsers/XMLConstants.java [hunk 4]
```diff
@@ -15,62 +15,36 @@
     public static final String FATAL_DEVIATION_THRESHOLD = "fatalDeviationThreshold";
     public static final String WARNING_DEVIATION_THRESHOLD = "warningDeviationThreshold";
```
Comment (dlatnikov) [inline chassis/spring.schema/src/main/java/com/griddynamics/jagger/xml/beanParsers/XMLConstants.java]: these constants together with properties below are useless now
```

 <property name="fatalDeviationThreshold"
                          value="${chassis.engine.e1.reporting.session.comparison.decision.maker.fatal.deviation.threshold}"/>
                <property name="warningDeviationThreshold"
                          value="${chassis.engine.e1.reporting.session.comparison.decision.maker.warning.deviation.threshold}"/>
```

File: chassis/configuration/configuration/reporter/session.comparison.conf.xml (hunk 3 above)
Comment (anton-antonenko) [inline chassis/configuration/configuration/reporter/session.comparison.conf.xml]: Should it be removed?
I see that sessionComparator is used in reporting.conf.xml

File: chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/reporting/DefaultSessionStatusDecisionMaker.java [hunk 5]
```diff
@@ -31,24 +30,12 @@
 
 public class DefaultSessionStatusDecisionMaker implements SessionStatusDecisionMaker {
     private static final Logger log = LoggerFactory.getLogger(DefaultSessionStatusDecisionMaker.class);
-    private final double EPSILON = 0.0000001;
     private double successRateThreshold = 1.0;
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/reporting/DefaultSessionStatusDecisionMaker.java]: remove as well is is not used

\+ remove usage of setter for this parameter and corresponding property: chassis.master.reporting.successrate.threshold=1.0

File: chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/reporting/DefaultSessionStatusDecisionMaker.java [hunk 6]
```diff
@@ -31,24 +30,12 @@
...
-    // key to avoid additional verification of success rate to set status of test
-    // recommended way to use limits based decision making for this purpose
-    private boolean doNotUseDeprecatedMethodToSetTestStatus = false;
-
     private String description;
 
     @Override
     public Decision decideOnTest(WorkloadTaskData workloadTaskData) {
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/reporting/DefaultSessionStatusDecisionMaker.java]: please leave comment
// ignore this decision maker

**Pull request merged**